Branch-SDK/src/io/branch/referral/ServerRequest.java

```diff
@@ -182,6 +185,9 @@ public JSONObject getPostWithInstrumentationValues(ConcurrentHashMap<String, Str
             }
         } catch (JSONException ignore) {
         }
+        catch (ConcurrentModificationException ex){
+            extendedPost = params_;
```

Comment:
so if we detect that the keys to `params_` were in fact modified, we take those values and assign to extendedPost?